src/tests/EdFi.DataManagementService.Tests.E2E/Hooks/SetupHooks.cs

```diff
@@ -9,21 +9,16 @@
 namespace EdFi.DataManagementService.Tests.E2E.Hooks;
 
 [Binding]
-public class SetupHooks
+public static class SetupHooks
 {
     private static ContainerSetupBase? _containerSetup;
 
-    private static bool _useTestContainers = false;
-    private static bool _openSearchEnabled = false;
-
     [BeforeTestRun]
     public static async Task BeforeTestRun()
     {
-        _useTestContainers = AppSettings.UseTestContainers;
-        _openSearchEnabled = AppSettings.OpenSearchEnabled;
-
-        if (_useTestContainers)
-            if (_openSearchEnabled)
+        if (AppSettings.UseTestContainers)
+        {
+            if (AppSettings.OpenSearchEnabled)
```

Comment:
Simpler to use the `AppSettings` directly instead of mapping to a local variable.